src/grt/grt-avls.adb

```diff
@@ -217,7 +217,7 @@ package body Grt.Avls is
          Internal_Error ("avls.get_node");
       end if;
       Insert (Tree, Cmp, N, AVL_Root, Res);
-      Check_AVL (Tree, AVL_Root);
+      -- Check_AVL (Tree, AVL_Root);
```

Comment:
To be replaced with: `pragma Debug(Check_AVL(...));`, so that the check is kept in case of debug build.

Reply:
Done. Out of curiosity, how do I enable debug build?